packages/ember-routing/lib/system/router.js

```diff
@@ -865,6 +877,10 @@ function triggerEvent(handlerInfos, ignoreFailure, args) {
       if (handler.actions[name].apply(handler, args) === true) {
         eventWasHandled = true;
       } else {
+        // Should only hit here if a non-bubbling error action is triggered on a route.
+        if (name === 'error') {
+          markErrorAsHandled(args[0]);
```

Comment:
args[0] is the error. It's the first argument to the error action. Should I call it out as a var?